kiwi_lint/__init__.py

```diff
@@ -40,3 +41,4 @@ def register(linter):
     linter.register_checker(OneToOneFieldChecker(linter))
     linter.register_checker(ClassBasedViewChecker(linter))
     linter.register_checker(DatetimeChecker(linter))
+    linter.register_checker(GenericForeignKeyChecker(linter))
```

Comment:
Rebase to latest master and include this after `FormFieldChecker` (e.g. at the end)